test/assemble.js

```diff
@@ -4,19 +4,49 @@ const { expect } = require('chai')
 const { assemble } = require('..')
 
 describe('assemble', () => {
-  const xfrms = {
-    foo: add(1),
-    bar: {
-      baz: add(2)
-    },
-    bat: 1
-  }
-
-  it('assembles the result of multiple transforms into a new object', () =>
-    expect(assemble(xfrms, 1)).to.eql({ foo: 2, bar: { baz: 3 }, bat: 1 })
-  )
-
-  it('is curried', () =>
-    expect(assemble(xfrms)(1)).to.eql({ foo: 2, bar: { baz: 3 }, bat: 1 })
-  )
+  describe('unary', () => {
+    const xfrms = {
+      foo: add(1),
+      bar: {
+        baz: add(2)
+      },
+      bat: 1
+    }
+
+    it('assembles the result of multiple transforms into a new object', () =>
+      expect(assemble(xfrms, 1)).to.eql({ foo: 2, bar: { baz: 3 }, bat: 1 })
+    )
+
+    it('is curried', () =>
+      expect(assemble(xfrms)(1)).to.eql({ foo: 2, bar: { baz: 3 }, bat: 1 })
+    )
+  })
+
+  describe('n-ary', () => {
+    const sjoin = glue => (...params) => params.join(glue)
```

Comment:
I got hung up on the word `glue` until I realized it was just the separator. 😄

Reply:
[Sometimes, i return to my humble PHP roots](http://php.net/manual/en/function.implode.php)